src/net/sourceforge/kolmafia/swingui/panel/ItemManagePanel.java

```diff
@@ -370,7 +372,7 @@ protected int getDesiredItemAmount(
       final String itemName,
       final int itemCount,
       final String message,
-      final int quantityType) {
+      final QuantityType quantityType) {
     int quantity = 0;
     switch (quantityType) {
```

Comment:
Probable out of scope for this simple change, but making this an enum might allow this to be a switch expression.

There are other issues - break at the end of new-style switch block, instanceof followed by casts, etc. - but those sre not related to enumitude.